plenoptic/simulate/models/frontend.py

```diff
@@ -34,9 +30,7 @@ class Front_End(nn.Module):
 
     TODO
     -----
-    rerange input in [0,1]
-    make 12 parameters explicitely learnable
-    adapt to image size
+    Parameterize Conv2D as Difference of Gaussians -- the model should have 13 params total
```

Comment:
I'm not a big fan of TODOs in comments: I feel like we forget about them and they quickly get out of date, and only the author knows to check for them. Can we open an issue for it instead?

Reply:
Agreed. Turned into Issue #88